plasoscaffolder/lib/definitions.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+"""The format specification classes."""
+
+PROJECT_UNDEFINED = 'undefined'
```

Comment:
Do we need a central definition of these for some reason? Shouldn't they just live in the project classes?